pages/watering.js

```diff
@@ -0,0 +1,59 @@
+import WateringSchedule from "@/components/WateringSchedule";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import BackArrow from "@/components/MyPlant/BackArrow";
+
+function useDayCount(startDay) {
+  const [day, setDay] = useState(startDay);
+  function handleNextDay() {
+    setDay(day + 1);
+  }
+  return [day, handleNextDay];
+}
+
+export default function ReminderPage({ plants }) {
+  const [plantsToWater, setPlantsToWater] = useState([]);
+
+  const [day, nextDay] = useDayCount(0); // only for simulation
+
+  const myPlants = plants.filter((plant) => plant.isOwned);
+
+  const daysOfWeek = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
```

Comment:
Consider to move this outside your component. Although the performance gain might be negligible in this specific case, moving static data outside of a component prevents it from being recreated on every render.